Common/ChallengeMode/ChallengeModeGlobalNPC.cs

```diff
@@ -7,6 +7,13 @@
 
 namespace BossRush.Common.ChallengeMode {
 	internal class ChallengeModeGlobalNPC : GlobalNPC {
+		public override void SetDefaults(NPC entity) {
+			//int amount = BossRushUtils.AmountOfModCurrentlyEnable();
+			//entity.lifeMax *= amount;
+			//entity.life = entity.lifeMax;
+			//entity.damage *= amount;
+			//entity.defense *= amount;
```

Comment:
Please put `// TODO: ...` if you gonna do it later. Otherwise, remove code.

Reply:
got it